lib/chargify_api_ares/response_helper.rb

```diff
@@ -1,5 +1,19 @@
 module Chargify
   module ResponseHelper
+    def save
```

Comment:
The reason why `Subscription` implements uniqueness on it's own is because there are other overrides in the save method. I could move the uniqueness behavior to base and that would probably work out alright. That would also take care of the `Subscription` implementation by calling super.